nemo_skills/dataset/aime24/__init__.py

```diff
@@ -11,11 +11,11 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from nemo_skills.evaluation.metrics import MathMetrics
+from nemo_skills.evaluation.metrics.utils import get_metrics
```

Comment:
Let's always import without .utils like in evaluation.md

Reply:
Done